started two cube auto on aligned scale

christiancruzfhs opened the pull request.

christiancruzfhs committed:
started two cube auto on aligned scale

christiancruzfhs committed:
added intake after lowering elevator

Gabriel-J-Young committed:
changed the firstCubeFromGround heading from null to 270.

Modified the command to move the elevator and wrist down after the robot scoots back from the first scale scoring attempt but before the robot moves to pick up the second cube.

Modified the command to replicate the scooting back and forth behavior for the second scoring in auto from the first scoring in auto.

Gabriel-J-Young committed:
Merge branch 'master' into feature/two-cube-auto-start

JohnGilb commented on Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java:
```diff
@@ -0,0 +1,92 @@
+        this.addSequential(firstCubeFromGround);
+        
+        // intake block
+        this.addSequential(intake);
```
intake should probably be in parallel with driving towards the cube, and also it has to end after some time

Gabriel-J-Young replied:
done

JohnGilb commented on Competition2018/src/main/java/competition/commandgroups/ScoreOnAlignedScaleFromRightComandGroup.java:
```diff
@@ -0,0 +1,92 @@
+        this.addSequential(intake);
+
+        // Get in position
+        this.addSequential(pursuit);
```
We need a new movement to get back to to scoring position

Gabriel-J-Young replied:
done

Gabriel-J-Young committed:
added a command in the gripper subsystem that intakes for a set number of seconds.
Used new command in the ScoreOnAlgniedScaleFromRightCommandGroup when the robot intakes a cube.

Gabriel-J-Young committed:
I'm trying to fix checkstyle! yay!

Gabriel-J-Young committed:
I pressed the format button a few times.

Gabriel-J-Young committed:
fixed a typo

JohnGilb commented:
Integrated manually in other branches.
